crates/full-node/sov-sequencer/src/preferred/db/mod.rs

```diff
@@ -36,13 +36,28 @@ use crate::preferred::{exit_rollup, track_in_progress_batch_size};
 #[derive(Debug)]
 pub(crate) enum DbReadOutcome<T> {
     Success(T),
-    AbortedBecauseReplica,
+    AbortedBecauseReplica { db_replica: Option<String> },
```

Comment:
This is going to have the leader ID, right? So maybe
```suggestion
    AbortedBecauseReplica { db_leader: Option<String> },
```

Reply:
fixed